package/rmfakecloud-proxy/rmfakecloud-proxy.service

```
@@ -0,0 +1,12 @@
+[Unit]
+Description=rmfakecloud reverse proxy
+Documentation=https://github.com/ddvk/rmfakecloud-proxy
+After=home.mount
```

We probably should ensure that this is running before xochitl is when using the standard mechanism. It also means that xochitl will be stopped before this is on shutdown. This will avoid potential timing problems.
```suggestion
After=home.mount
Before=xochitl.service
```

---

It might also make sense to have it start before `launcher.service`

---

Good point, thanks! I also added `After=network.target`.